build/musicbrainz/Dockerfile

```diff
@@ -34,7 +34,7 @@ RUN git clone --depth=1 --branch $MUSICBRAINZ_VERSION https://github.com/metabra
 
 WORKDIR /musicbrainz-server
 
-ARG NODE_VERSION=8.11.3
+ARG NODE_VERSION=8.17.0
```

Comment:
Oops, v8 is no longer maintained according to https://nodejs.org/en/about/releases/ and MusicBrainz Server switched to v10 a long time ago. Let’s update to [v10.19.0](https://nodejs.org/en/blog/release/v10.19.0/) directly.